config/thoth.yaml

```diff
@@ -112,15 +112,6 @@ repositories:
         configuration:
           labels: [bot]
       - name: info
-      - name: version
```

Comment:
we dont need it because its not a library?

Reply:
exactly